# Frequency in Hz
float64 frequency
---

